R/mod_tableGen.R

```diff
@@ -274,6 +274,36 @@ mod_tableGen_server <- function(input, output, session, datafile = reactive(NULL
     session$sendCustomMessage("all_cols", all_cols)
   })
   
+  AVALS <- reactive({
+    req(datafile())
+    req("ADVS" %in% names(datafile()))
+    req(any(c("ATPT") %in% colnames(datafile()$ADVS)))
+    
+    avals <- datafile()$ADVS %>%
```

Comment:
This solution is very focused on the `ADVS` data (which is great for a POC), but there may be occasions when ATPT exists in other ADaMs... perhaps we should generalize this PR to perform these actions on any data set containing ATPT. Or if you'd rather open a new PR to generalize, that works too.

Reply:
I think it makes more sense to put the leg work in on this PR instead of pushing a half solution. Will just take a bit more work to generalize.